client/src/dateUtils.js

```diff
@@ -7,6 +7,9 @@ export const ON = "3"
 export const LAST_DAY = -1 * 1000 * 60 * 60 * 24
 export const LAST_WEEK = LAST_DAY * 7
 export const LAST_MONTH = LAST_DAY * 30
+export const LAST_3_MONTHS = LAST_MONTH * 3
+export const LAST_6_MONTHS = LAST_3_MONTHS * 2
+export const LAST_YEAR = LAST_6_MONTHS * 2
```

Comment:
I'd simplify these calculations. Now I mentally have to compute "last year is twice last 6 months, which is twice last 3 months, with is three times last month". I'd prefer:
```javascript
export const LAST_3_MONTHS = LAST_MONTH * 3
export const LAST_6_MONTHS = LAST_MONTH * 6
export const LAST_YEAR = LAST_MONTH * 12
```